packages/plugin-page-view-tracking-browser/test/page-view-tracking.test.ts

```diff
@@ -290,6 +290,24 @@ describe('pageViewTrackingPlugin', () => {
     });
   });
 
+  describe('teardown', () => {
```

Comment:
In a recent session fix for mobile, sometimes `onExitForeground` would not get called (crash, etc). Do you see any potential for a similar issue here? 

Is there any harm in calling `teardown()` in `setup()` to be safe?

Reply:
Discussed in Slack.

<img width="719" alt="image" src="https://github.com/amplitude/Amplitude-TypeScript/assets/5790872/13a02cfe-f8dd-4f7d-a00d-07e9d5f668bf">